README.md

```diff
@@ -25,7 +24,7 @@ Same as in Node.js, you just have to [browserify](https://github.com/substack/js
 
 Make the [multihashes.min.js](/dist/multihashes.min.js) available through your server and load it using a normal `<script>` tag, this will export the `multihashes` constructor on the `window` object, such that:
 
-```JavaScript
+```javascript
```

Comment:
Well, the weird capitalization is the name of the language
https://en.wikipedia.org/wiki/JavaScript
although yeah, I do see it seem written with all lowercase sometimes